system-int-test/test/cmr/system_int_test/admin/admin_permissions_test.clj

```diff
@@ -20,7 +22,9 @@
   ([url method token body]
    (let [response (client/request {:url url
                                    :method method
-                                   :body body
+                                   :accept "application/json"
+                                   :content-type mt/form-url-encoded
```

Comment:
The content-type is wrong. How the revision-date range is passed to the new route is confusing. From the code, it is passed in via query params; from the test, it appears to be in the request body. Either way would be fine, but they need to match. 

Even though JSON request body is more suitable with POST, I would like the revision-date-range to be passed in via query params -- it would be easier to use, consistent to existing CMR admin routes and logging will capture the parameters (not if they are in the request body).

Reply:
+1 on the point of consistency